pkg/util/docker/dockerfile/builder/client.go

```diff
@@ -17,6 +17,8 @@ import (
 	"github.com/fsouza/go-dockerclient/external/github.com/docker/docker/pkg/archive"
 	"github.com/fsouza/go-dockerclient/external/github.com/docker/docker/pkg/fileutils"
 	"github.com/golang/glog"
+
+	"github.com/openshift/origin/pkg/cmd/util/imageprogress"
```

Comment:
Unfortunately you can't take this dependency because we are going to move the builder out of our repo.  You can put image progress as a sub package of this code though.